### ISSUE TITLE
Выписка SBER_CREDIT_2409 некорректно обрабатывается при двухстрочной категории

### ISSUE THREAD

**Issue by ilabutin** [Ev2geny/Sberbank2Excel#56]
В выписке по кредитной карте из-за дополнительной информации название категории выводится в две строки:
![image](https://github.com/user-attachments/assets/39fe03e5-f69c-4c6e-bd15-c9a151f4bcb5)

В .txt файле это выглядит как:
```
24.09.2024	22:34	895977	Коммунальные платежи, связь, интернет. • Сниженная	5 226,26	999 999,99
ставка – 24.8%
26.09.2024	Kurgan OICO WEB
26.09.2024	13:56	429010	Рестораны и кафе • Сниженная ставка – 24.8%	419,00	888 888,88
26.09.2024	Gorod Moskva TEREMOK
```

И из-за второй строки похоже регулярное выражение для individual_entries не срабытывает и не находит такие транзакции.

**Comment by Ev2geny** [comment]
Да, с таким вариантом трансакции SBER_CREDIT_2409, действительно, не справится.
Спасибо, что сообщили.
А можете создать полную [анонимизированную выписку](https://github.com/Ev2geny/Sberbank2Excel#%D0%BF%D1%80%D0%B8%D0%BB%D0%BE%D0%B6%D0%B5%D0%BD%D0%B8%D0%B5-%D0%B1-%D0%BA%D0%B0%D0%BA-%D0%B1%D0%B5%D0%B7%D0%BE%D0%BF%D0%B0%D1%81%D0%BD%D0%BE-%D0%BF%D0%B5%D1%80%D0%B5%D1%81%D1%8B%D0%BB%D0%B0%D1%82%D1%8C-%D0%BF%D1%80%D0%BE%D0%B1%D0%BB%D0%B5%D0%BC%D0%BD%D1%8B%D0%B9-%D1%84%D0%B0%D0%B9%D0%BB) с такой трансакцей?

**Comment by Ev2geny** [comment]
Релиз [4.10.3](https://github.com/Ev2geny/Sberbank2Excel/releases/tag/v.4.10.3) должен работать с такой выпиской
